src/modules/SX126x/SX126x.cpp

```diff
@@ -1395,6 +1392,11 @@ float SX126x::getFrequencyError() {
 
 size_t SX126x::getPacketLength(bool update) {
   (void)update;
```

Comment:
This is no longer necessarry, the only reason this line was here originally was to prevent an unused argumetn warning.

Reply:
Removed in last commit.